production_configs/psf_tuned_zen40/lstmcpipe_config.yml

```diff
@@ -0,0 +1,44 @@
+## MC PRODUCTION FULL PIPELINE CONFIGURATION ##
+#
+# Template example to request a MC prod.
+
+
+workflow_kind: lstchain  # Not expected to be changed by the user
+prod_id: zen40az180_dl1ab_tuned_psf        # ex: local_no_n_islands. Default; v00 (if key left empty or None)
+
+base_path: /fefs/aswg/data/mc  # Path for centralised LST MC prods
+
+# Only for dl1 reprocessing - you will also need to change:
+# `r0_to_dl1` for `dl1ab` in "stages_to_be_run"
+dl1_reference_id: 20220215_v0.9.1_prod5_trans_80_local_tailcut_8_4
```

Comment:
The `dl1_reference_id` should be: `20220304_v0.9.3_prod5_trans_80_zen40az180_local_tailcut_8_4`

There wasn't any zen40deg MC prod analysed with v>0.8 up to now, and `20220215_v0.9.1_prod5_trans_80_local_tailcut_8_4` was referring a zen20deg prod (it wasn't written on the prod-id, no we will start to include it)